### ISSUE TITLE
Error: Spawn xdg-open ENOENT

### ISSUE THREAD

**Issue by sterumbelow** [homebridge-simplisafe3/homebridge-simplisafe3#259]
### Describe The Bug:

Trying to login using the new command line method. After following the link, logging in and copying the 'com.' link back in, I get an error. 

Error: spawn xdg-open ENOENT

### To Reproduce:

To reproduce this error, all I have to do is follow the instructions in the readme, trying to login with the new auth method. 

### Expected behavior:

To log me in.

### Environment:

* **Plugin version**: 
Latest. 
* **Node.js Version**: 
* **NPM Version**: 
* **Homebridge Version**:  latest Homebridge. 
* **Operating System**:  RaspberryPi. 

### Homebridge Config:

```json
Paste the relevant section of your homebridge config.json here
```

### Logs:

[Link to logs]()
```
Paste Homebridge Logs here
```

**Comment by sterumbelow** [comment]
Just found the button in settings, which allowed me to paste the URL across. 

Welcome to close this issue, although I still couldn't get it to work solely through terminal.

**Comment by slichtenthal** [comment]
I was able to get past this issue by manually installing xdg-open running the command sudo apt-get install xdg-utils.  I still get another error after however, “exited with code 3”

**Comment by shamoon** [comment]
What Pi are you using? I was thinking I'd pick one up to test this out but holy cow prices are so inflated right now for them. Chip shortage I guess =/

**Comment by shamoon** [comment]
Addressed in v1.8.6
